bin/test_flakiness

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+
+# Determine how flaky our UI tests by measuring recent failure rate
+# using data from the Saucelabs API. Prints out every UI test from
+# least flaky to most flaky.
+
+require_relative '../deployment'
+require 'rest-client'
+
+username = ENV['SAUCE_USERNAME'] || CDO.saucelabs_username
+key = ENV['SAUCE_ACCESS_KEY'] || CDO.saucelabs_authkey
+
+PER_REQUEST = 1500 # maximum returned per API call (undocumented)
+NUM_REQUESTS = 50 # rate limit: 15 request/s with 300 request burst https://wiki.saucelabs.com/display/DOCS/Rate+Limits+for+the+Sauce+Labs+REST+API
```

Comment:
50 is less than 300 so calling this script once with these numbers will not exceed the burst rate limit. Left as an exercise for the reader to not run this script 6 times in a row? (I think the limits are per-user not per-account, so you running it too much will just make you wait a minute, and not break the test run or my account, for example)